addons/parachute/functions/fnc_storeParachute.sqf

```diff
@@ -0,0 +1,27 @@
+#include "script_component.hpp"
+/*
+ * Author: joko // Jonas
+ * Add the Reserve Parachute to Units or Save Backpack if is a Parachute in Unit
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * 0: Unit
+ * 1: getAllGear-Array
+ *
+ * Example:
+ * None
+ *
+ * Public: No
+ */
+private ["_unit","_backpack"];
+_unit = _this select 0;
+_backpack = (_this select 1) select 6 ;
+if ((vehicle _unit) isKindOf "ParachuteBase" || (backpack _unit == "")) then {
+    _unit addBackpack (_unit getVariable[QGVAR(backpackClass),"ACE_NonSteerableParachute"]);
```

Comment:
@jokoho48, the condition above is incomplete. This would add a non steerable parachute every time e.g. you reload your weapon without a packpack.

(`playerInventoryChanged` fires every time the inventory of the unit changes for whatever reason, including loading magazines on the gun, dropping things from the ground, etc.)